docs/tools/vdb_table/data/azureai.json

```diff
@@ -59,22 +61,22 @@
   },
   "bm25": {
     "support": "full",
-    "source_url": "https://learn.microsoft.com/en-us/azure/search/index-ranking-similarity",
-    "comment": "https://learn.microsoft.com/en-us/azure/search/index-ranking-similarity"
+    "source_url": "https://learn.microsoft.com/azure/search/index-ranking-similarity",
+    "comment": ""
   },
   "full_text": {
     "support": "none",
-    "source_url": "",
+    "source_url": "https://learn.microsoft.com/azure/search/search-query-overview#types-of-queries",
     "comment": ""
   },
   "embeddings_text": {
     "support": "full",
-    "source_url": "https://learn.microsoft.com/en-us/azure/search/cognitive-search-custom-skill-web-api",
-    "comment": "Custom Web API skill in skillsets - Azure AI Search | Microsoft Learn https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/announcing-the-public-preview-of-integrated-vectorization-in/ba-p/3960809"
+    "source_url": "https://learn.microsoft.com/azure/search/cognitive-search-custom-skill-web-api",
```

Comment:
Wouldn't it better to point at the integrated vectorization option? e.g. https://learn.microsoft.com/en-us/azure/search/vector-search-integrated-vectorization